x-pack/plugins/security_solution/public/timelines/components/timeline/tabs_content/index.tsx

```diff
@@ -352,7 +355,13 @@ const TabsContentComponent: React.FC<BasicTimelineTab> = ({
 
   const setSecurityAssistantAsActiveTab = useCallback(() => {
     setActiveTab(TimelineTabs.securityAssistant);
-  }, [setActiveTab]);
+    if (activeTab !== TimelineTabs.securityAssistant) {
+      reportAssistantInvoked({
```

Comment:
I tried to do all of the telemetry from the package, but the rendering in timeline proved difficult and I did not want a bunch of consumer related code (ie: `isTimeline`) in the assistant package. Therefore, this is the sole EBT tracker  called from security solution for assistant